ros_ws/src/common/constants.h

```diff
@@ -59,6 +67,10 @@ const std::string TELEOP_NODE = "teleop_control";
  * @brief The node name for the teleop_mux node
  */
 const std::string TELEOP_MUX_NODE = "teleop_mux_node";
+/**
+ * @brief The node name for the motor_controller node
+ */
+const std::string MOTOR_CONTROLLER_NODE = "motor_controller_node";
```

Comment:
It probably makes sense for us to separate the node names into their own file and put them in a namespace. This doesn't have to happen now for this pull request, but it should happen sometime in the near future.